MTM-44126 Added related topics to Getting started (#613)
SoftwareAG/c8y-docs: content/users-guide/getting-started-bundle/platform-applications.md

  - /users-guide/getting-started/#cumulocity-applications
---

{{< c8y-admon-related >}}

> - [Administration > Managing applications](/users-guide/administration/#managing-applications) for more information on managing applications and microservices in {{< product-c8y-iot >}}.

Reviewer: ```suggestion
- [Administration > Managing applications](/users-guide/administration/#managing-applications) for details on the available applications and information on managing applications in {{< product-c8y-iot >}}.
```
